### ISSUE TITLE
Make SegmentedBarItem's usable in markup

### ISSUE THREAD

**Issue by hdeshev** [NativeScript/nativescript-angular#173]
Add support for child tags in SegmentedBar (defining without a binding):

```xml
<SegmentedBar row="1" colspan="3" class="toggles" (selectedIndexChanged)="filter()" selectedBackgroundColor="#B76769">
    <SegmentedBarItem title="All"></SegmentedBarItem>
    <SegmentedBarItem title="Active" completed="false"></SegmentedBarItem>
    <SegmentedBarItem title="Done" completed="true"></SegmentedBarItem>
</SegmentedBar>
```

Related discussion: #80

**Comment by nsndeck** [comment]
Closed with commit #179.

**Comment by dotdreaming** [comment]
Can the <SegmentedBarItem> take a [nsRouterLink]="['/second']" attribute and be used with the <route-outlet> router (v3)?
`<SegmentedBar row="1" colspan="3" class="toggles" (selectedIndexChanged)="filter()" selectedBackgroundColor="#B76769">
    <SegmentedBarItem title="All" [nsRouterLink]="['/first']" ></SegmentedBarItem>
    <SegmentedBarItem title="Active" completed="false" [nsRouterLink]="['/second']" ></SegmentedBarItem>
    <SegmentedBarItem title="Done" completed="true" [nsRouterLink]="['/third']" ></SegmentedBarItem>
</SegmentedBar>`
